docs-ref-conceptual/microsoft-graph-migration.md

```diff
@@ -14,15 +14,15 @@ keywords: microsoft graph, ms graph, active directory graph, ad graph
 
 # Microsoft Graph migration
 
-Due to [the deprecation of Azure Active Directory (Azure AD) Graph](/graph/migrate-azure-ad-graph-overview), the underlying Active Directory Graph API will be replaced by [Microsoft Graph API](/graph/api/overview) in Azure CLI 2.37.0.
+Due to [the deprecation of Azure Active Directory (Azure AD) Graph](/graph/migrate-azure-ad-graph-overview), the underlying Active Directory Graph API replaces [Microsoft Graph API](/graph/api/overview) in Azure CLI 2.37.0.
```

Comment:
This totally changes the meaning of this sentence thus making it wrong.

Reply:
I personally don't think forbidding the use of passive voice is a good idea. Passive voice is perfect for representing **changes**, as its words are in chronological order: AD Graph -> Microsoft Graph: AD Graph is replaced by Microsoft Graph.

Reply:
I respect your opinion on active/passive voice, @Jiasli, but that is one MSFT guideline that is not likely to change.  A "is replaced by" works as we are now in azCLI 2.53.0.  Please see PR [3381](https://github.com/MicrosoftDocs/azure-docs-cli/pull/3881) for correction.